content/en/docs/tasks/administer-cluster/sysctl-cluster.md

```diff
@@ -53,6 +53,14 @@ same node. This means that setting a _safe_ sysctl for one pod
 - must not allow to gain CPU or memory resources outside of the resource limits
   of a pod.
 
+{{< note >}}
+Starting from Kubernetes version 1.23, kubelet supports the use of "/" or "." as separators for sysctl.
+Sysctls can be converted to a slash as a separator.
+For example, the sysctl `kernel.shm_rmid_forced` can also be written as `kernel/shm_rmid_forced` with a slash as a separator.
```

Comment:
```suggestion
For example, you can represent the same sysctl name as `kernel.shm_rmid_forced` using a
period as the separator, or as `kernel/shm_rmid_forced` using a slash as a separator.
```

Reply:
Thanks for your suggestion, I will update it.